eng/scripts/Create-APIView.ps1

```diff
@@ -185,18 +213,18 @@ function New-SwaggerAPIViewTokens {
   }
   LogGroupEnd
 
-  # Get Related Swagger ReadMe Files
-  $swaggerReadMeFiles = [System.Collections.Generic.HashSet[string]]::new()
+  # Get Related AutoRest Configuration Information
+  $autoRestConfigInfo = [System.Collections.Generic.Dictionary[string, object]]::new()
   $changedSwaggerFiles | ForEach-Object {
-    $readmeFile = Get-SwaggerReadMeFile -swaggerFile $_
-    if ($readmeFile) {
-      $swaggerReadMeFiles.Add($readmeFile) | Out-Null
+    $configInfo = Get-AutoRestConfigInfo -swaggerFile $_
+    if ($null -ne $configInfo -and -not $autoRestConfigInfo.ContainsKey($configInfo.ConfigPath)) {
+      $autoRestConfigInfo[$configInfo.ConfigPath] = $configInfo
```

Comment:
Why make this a dictionary instead of just a list?

Reply:
I guess the only advantage of dictionary is that it's easier to avoid duplicates. I was using set earlier them switched to dictionary. You are right it could just be a list.